packages/build/src/tracing/main.ts

```diff
@@ -1,6 +1,5 @@
+import { HoneycombSDK } from '@honeycombio/opentelemetry-node'
```

Comment:
The `HoneycombSDK` provides a couple of things out the box which make the initialisation simpler.